llama2.mojo

```diff
@@ -980,10 +975,9 @@ fn main() raises:
 
         # Advance forward
         token = next_token
-        pos += 1
 
         if start == 0:
             start = time_in_ms()
 
     let end = time_in_ms()
-    print("\nachieved tok/s: ", (pos - 1) / (end - start) * 1000)
+    print("\nachieved tok/s: ", (steps - 1) / (end - start) * 1000)
```

Comment:
Why count steps instead of tokens?  Any benchmarking probably needs this line reverted.

Reply:
ah of course... sorry for the waste of time, i believed too much in the power of alias and thought there was a speed up lol that explains it